subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -32,14 +32,22 @@
 
 #define LAST_VALID_CIS_HANDLE (CONFIG_BT_CTLR_CONN_ISO_STREAMS + LL_CIS_HANDLE_BASE - 1)
 
+static int init_reset(void);
```

Comment:
There seems to be quite a lot of code being moved around in this file, which makes it near-impossible to determine what is moved and what has changed, so I decided not to go into detail of the changes in this file.

Reply:
The re-order was to align the order of functions as followed by other state/role implementation function.The order is to place the functions in the order of actual execution follow.

I hope @asmk-ot, @mtpr-ot  and team could take a look on the actual functional changes.